ext/web/13_message_port.js

```diff
@@ -85,16 +86,37 @@ const MessageChannelPrototype = MessageChannel.prototype;
 const _id = Symbol("id");
 const MessagePortIdSymbol = _id;
 const _enabled = Symbol("enabled");
+const _nodeWorker = Symbol("nodeWorker");
+const _closeEventDispatched = Symbol("closeEventDispatched");
 
 /**
  * @param {number} id
  * @returns {MessagePort}
  */
-function createMessagePort(id) {
+function createMessagePort(id, nodeWorker = false) {
+  if (nodeWorker) {
+    defineEventHandler(MessagePort.prototype, "close");
+  }
```

Comment:
We should remove this - we don't need `onclose` handler - just emitting `close` event for Node compat. Can be achieved without this.

Reply:
node supports `on("close", handler)` on a worker so I introduced it - am I missing something?

Reply:
`onclose` is "Web" way, while `on("close", ..)` is the Node way. If you do it this way, then suddenly `MessagePort.onclose` becomes a thing, which is not available in the https://developer.mozilla.org/en-US/docs/Web/API/MessagePort#events